crates/ruma-client-api/src/profile.rs

```diff
@@ -52,6 +52,16 @@ impl StaticProfileField for DisplayName {
     const NAME: &str = "displayname";
 }
 
+/// The user's time zone.
+#[derive(Debug, Clone, Copy)]
+#[allow(clippy::exhaustive_structs)]
+pub struct TimeZone;
```

Comment:
Nit: Maybe we should move these profile field structs to their own private module (to be re-exported here) that uses `#[allow(clippy::exhaustive_structs)]` at the module level?

Reply:
All right, done!